src/Activation/AcceptEula.tsx

```diff
@@ -1,77 +1,50 @@
-import React, { useEffect, useState, FunctionComponent } from "react"
+import React, { useState, FunctionComponent } from "react"
+import env from "react-native-config"
 import {
   TouchableOpacity,
   Linking,
   StyleSheet,
   View,
   SafeAreaView,
-  ActivityIndicator,
 } from "react-native"
 import { useTranslation } from "react-i18next"
-import loadLocalResource from "react-native-local-resource"
-import WebView, { WebViewNavigation } from "react-native-webview"
 import { useNavigation } from "@react-navigation/native"
 import { SvgXml } from "react-native-svg"
 
 import { Icons } from "../assets"
+import { GradientBackground } from "../components"
 import { GlobalText } from "../components/GlobalText"
 import { ActivationScreens } from "../navigation"
 import { Button } from "../components/Button"
-import enEulaHtml from "../locales/eula/en.html"
-import esPREulaHtml from "../locales/eula/es_PR.html"
-import htEulaHtml from "../locales/eula/ht.html"
 
-import { Forms, Iconography, Colors, Spacing, Outlines } from "../styles"
+import {
+  Forms,
+  Iconography,
+  Colors,
+  Spacing,
+  Outlines,
+  Typography,
+  Layout,
+} from "../styles"
 import { useStatusBarEffect } from "../navigation"
-
-const LoadingIndicator = () => {
-  return (
-    <View style={style.loadingIndicator}>
-      <ActivityIndicator size={"large"} color={Colors.neutral100} />
-    </View>
-  )
-}
-
-const DEFAULT_EULA_URL = "about:blank"
-type AvailableLocale = "en" | "es_PR" | "ht"
+import arrow from "../assets/svgs/arrow"
 
 const AcceptEula: FunctionComponent = () => {
   useStatusBarEffect("dark-content")
-  const [html, setHtml] = useState<string | undefined>(undefined)
-  const [isLoading, setIsLoading] = useState(true)
   const [boxChecked, toggleCheckbox] = useState(false)
-  const {
-    t,
-    i18n: { language: localeCode },
-  } = useTranslation()
+  const { t } = useTranslation()
   const navigation = useNavigation()
 
-  const EULA_FILES: Record<AvailableLocale, string> = {
-    ["en"]: enEulaHtml,
-    ["es_PR"]: esPREulaHtml,
-    ["ht"]: htEulaHtml,
+  const handleOnPressNext = () => {
+    navigation.navigate(ActivationScreens.ActivateProximityTracing)
   }
-  const eulaPath = EULA_FILES[localeCode as AvailableLocale] || enEulaHtml
 
-  const shouldStartLoadWithRequestHandler = (
-    webViewState: WebViewNavigation,
-  ) => {
-    let shouldLoadRequest = true
-    if (webViewState.url !== DEFAULT_EULA_URL) {
-      Linking.openURL(webViewState.url)
-      shouldLoadRequest = false
-    }
-    return shouldLoadRequest
+  const linkToPrivacyPolicy = async () => {
+    await Linking.openURL(env.PRIVACY_POLICY_URL)
```

Comment:
Is this in our new configuration context?

Reply:
Looks like that just got merged an hour ago! Will pull & rebase